src/test/scala/nl.knaw.dans.easy.solr4files/CustomMatchers.scala

```diff
@@ -17,20 +17,21 @@ package nl.knaw.dans.easy.solr4files
 
 import java.io.File
 
-import scala.io.Source
+import org.scalatest.matchers.{ MatchResult, Matcher }
```

Comment:
I assume formatting issues in `CustomMatchers`, `DebugConfigSpec` and `ReadmeSpec` come from the archetype? Maybe also fix these over there.

Reply:
Formatting can trigger codacy warnings you might not want to trigger without solving it in the archetype and all other projects with the same code snippets.